brainstormforce/astra
# AST-2008 - Customizer: Woocommerce Cart Click Action "Cart Page"

### Description


-  New option in "Cart Click Action" -> "Cart Page"
- Upon activation, will redirect user to Cart Page
- Shortened other option names "Dropdown Cart" -> "Dropdown" and "Slide-In Cart" -> "Slide In" to accomodate space for new option.

### Screenshots

https://d.pr/v/XFA3AT

### Types of changes

- modified option, cart click action with new choice: 'Cart Page'.
- changes in markup, to redirect to cart page.







### How has this been tested?

-  In customizer > header builder > cart > cart click action
- check to see if a new option 'Cart Page' exists
- Click on it to activate
- now refresh website or in customizer preview, click on cart icon
- It should redirect to shopping cart page
- also tested with other options, "Dropdown", "Slide In", to check if changes do not break anything.

### Checklist:
